api/v1alpha1/clusterenvironment_types.go

```diff
@@ -20,24 +20,51 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // ClusterEnvironmentSpec defines the desired state of ClusterEnvironment
 type ClusterEnvironmentSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// The name of the ClusterEnvironment
+	Name string `json:"name"`
+
```

Comment:
Is there any reason we can't use the name of the resource itself for naming purposes?  IMO this is duplicate information.